### DISCUSSION TITLE
Linked Passwords

### DISCUSSION THREAD

**Discussion by jdevstart** [dani-garcia/vaultwarden#3361] [Ideas]
Wondering if there is a way to do this (and if not), if it is something that can be implemented.

I normally use my e-mail address to sign in to do administrative tasks so for example john_doe@company.com, this email address has an alias domain\jdoe.

They both use the same password and if one is changed, the other follows suite but unfortunately, some services do require the login to be specific.

My question is, is there a way to link passwords of different items?  This would be ideal for multiple items that use the same password but different login formats to sync up.

I'm taking advantage of the Custom fields as well but it's not the best approach.

**Comment by jjlin** [comment]
There would have to be client-side support for this, so it would have to be implemented upstream first. That probably won't happen anytime soon, but you can vote for

https://community.bitwarden.com/t/field-references-ability-to-set-different-usernames-for-different-websites-using-the-same-password/1624
